server/statsPageRedis.js

```diff
@@ -108,12 +131,18 @@ module.exports = function(app, env) {
         <!doctype html>
         <html>
           <head>
-            <meta charset="utf-8"/>
+            <meta charset="utf-8" name="viewport"
+                      content="height=device-height, 
+                      width=device-width, initial-scale=1.0, 
+                      minimum-scale=1.0, maximum-scale=1.0, 
+                      user-scalable=no, target-densitydpi=device-dpi">
             <title>AdaLite Wallet Stats</title>
             <link rel="icon" type="image/ico" href="assets/favicon.ico">
           </head>
           <body>
             Stats of transaction submissions and estimates of unique IPs visits per day, month and in total.
+            <a href="#tableOfContents" style="position:fixed;top:93%;left:calc(10% + 250px)">Table</a>
```

Comment:
wouldn't it be more sensible to use em instead of 250px? What about narrow screens?

Reply:
maybe it would, I haven't experimented with for that long. This solution works for screens with width >= 320px which seemed reasonable but I agree this solution is far from ideal. I 'll look into it.

Reply:
Using em instead of px didn't seem to make much of a difference, as mentioned the layout is still acceptable on 320px width devices which seems good enough to me.